### ISSUE TITLE
Variable descriptions for each dataset

### ISSUE THREAD

**Issue by SamPassmore** [EvolEcolGroup/pastclim#9]
Hi, 

Is there a table available that describes the variables within each dataset? Like with table 1 in the pre-print, but for all variables, and which we could access from within the package? (for plotting titles etc.)  

For example: what is mo_npp_[month]?

**Comment by dramanica** [comment]
Not at the moment, but it is a very good suggestion. I'll add something along those lines as soon as I have a minute (it's teaching term over here, so probably not until next week). In the mean time, mo_npp_[month] is monthly net primary productivity.

**Comment by SamPassmore** [comment]
FWIW I started this file for my own purposes. 

[pastclim_variabledescription.csv](https://github.com/EvolEcolGroup/pastclim/files/10027211/pastclim_variabledescription.csv)

**Comment by dramanica** [comment]
@SamPassmore: I have added the functionality you suggested in the `dev` branch. You will need to update
`pastclim` with:

`devtools::install_github("EvolEcolGroup/pastclim", ref="dev")`

Then, to get the table with additional details for the variables use:

`library(pastclim)`
`get_vars_for_dataset(dataset="Example", details = TRUE)`

For pretty labels, you can use `var_labels`:

```
climate_20k <- region_slice(
  time_bp = -20000,
  bio_variables = c("bio01", "bio10", "bio12"),
  dataset = "Example"
)
terra::plot(climate_20k, main = var_labels(climate_20k, dataset = "Example"))
```

**Comment by dramanica** [comment]
This functionality is now available in v1.2.1
